test/near_provider.test.js

```diff
@@ -468,16 +464,12 @@ describe('\n---- PROVIDER ----', () => {
 
             test('can deploy', withWeb3(async (web) => {
                 let zombies = new web.eth.Contract(zombieABI);
-                try {
-                    let result = await zombies
-                        .deploy({data: `0x${zombieCode}`})
-                        .send({from: web._provider.accountEvmAddress});
-                    expect(result._address).toBeDefined();
-                    expect(result._address.length).toStrictEqual(42);
-                    expect(result._address.slice(0, 2)).toStrictEqual('0x');
-                } catch (e) {
-                    return e;
-                }
+                let result = await zombies
```

Comment:
Ah I think this is the right track. We should really rewrite these where we expect the error to be thrown. Ok one moment. I will find the syntax pattern. Jest has a specific way it wants this iirc.